src/db.c

```diff
@@ -202,15 +203,15 @@ robj *lookupKeyWriteOrReply(client *c, robj *key, robj *reply) {
  * if the key already exists, otherwise, it can fall back to dbOverwrite. */
 static void dbAddInternal(serverDb *db, robj *key, robj *val, int update_if_existing) {
     dictEntry *existing;
-    int slot = getKeySlot(key->ptr);
-    dictEntry *de = kvstoreDictAddRaw(db->keys, slot, key->ptr, &existing);
+    int dbix = getKVStoreIndexForKey(key->ptr);
+    dictEntry *de = kvstoreDictAddRaw(db->keys, dbix, key->ptr, &existing);
```

Comment:
I don't think db-index is a good name for this. It makes me think of SELECT.

KVStoreIndex is fine though. Maybe `kvix` for the variable, or call it dict index? Kvstore has dict in many of its function names already.

Reply:
`kvix` sounds good to me.

Reply:
Maybe it could have been the name of Valkey? It's a bit like kivi and whatever we discussed. :D

Reply:
> It's a bit like kivi and whatever we discussed. :D

lol, that's true 🥝